test/dotnet-run.Tests/RunTests.cs

```diff
@@ -20,7 +20,7 @@ public void RunsSingleTarget()
             TestInstance instance = TestAssetsManager.CreateTestInstance(Path.Combine(RunTestsBase, "TestAppFullClr"))
```

Comment:
CC @schellap to confirm, but I don't think so.

And honestly, even if it were, that's not what your code is doing right now. You have that condition there, so, add a test to cover it. The fact you have that there means that someone is relying on it for things to work, without the test, we may get broken and not really know.